src/main/java/de/bonndan/nivio/input/LinksResolver.java

```diff
@@ -42,8 +42,11 @@ public void resolve(LandscapeDescription input) {
         LOGGER.info("Waiting for completion of {} external link handlers.", completableFutures.size());
         try {
             CompletableFuture.allOf(completableFutures.toArray(CompletableFuture[]::new)).get(2, TimeUnit.MINUTES);
-        } catch (InterruptedException | ExecutionException | TimeoutException e) {
+        } catch (ExecutionException | TimeoutException e) {
             processLog.error(new ProcessingException("Failed to complete all external data resolvers", e));
+        } catch (InterruptedException e) {
+            processLog.error(new ProcessingException("Failed to complete all external data resolvers", e));
+            Thread.currentThread().interrupt();
```

Comment:
Weil die Message dieselbe ist, ist das hier vielleicht schöner:

```suggestion
        } catch (InterruptedException | ExecutionException | TimeoutException e) {
            processLog.error(new ProcessingException("Failed to complete all external data resolvers", e));
            if (e instanceof InterruptedException) {
                Thread.currentThread().interrupt();
            }
```